Makefile

```
@@ -15,12 +16,21 @@ TESTING_CONFIG := $(CURDIR)/tests/data/testing_config.py
 build:
 	$(CONTAINER_ENGINE) build -f Containerfile -t $(IMAGE_NAME) .
 
-re-build:
+rebuild:
 	$(CONTAINER_ENGINE) build --no-cache --pull=true -f Containerfile -t $(IMAGE_NAME) .
 
 run:
-	$(CONTAINER_ENGINE) run -it --rm -v $(CURDIR):/config:z -v $(SECRETS_DIR):/secrets:z --env TOKMAN_CONFIG=/config/config.py --env WORKERS=$(WORKERS) --env LOG_LEVEL=$(LOG_LEVEL) --publish 8000 tokman
+	$(CONTAINER_ENGINE) run -it --rm -v $(CURDIR):/config:z -v $(SECRETS_DIR):/secrets:z --env TOKMAN_CONFIG=/config/config.py --env WORKERS=$(WORKERS) --env LOG_LEVEL=$(LOG_LEVEL) --publish 8000 $(IMAGE_NAME)
+
+build-test-image:
```

```suggestion
build-test-image: build
```

If the two images depend on each other, I think it makes sense to also express this dependency here.

---

The image proper and the test image are not dependent on each other.

---

The image proper and the test image are not dependent on each other.

---

OK, sorry, crossed in the mail, so to speak.

---

Should rebuild-test-image also depend on build (or rebuild)?

---

The semantically correct would be to depend on rebuild.